Sources/AppCoinsSDK/Domain/Repositories/Product/ProductRepository.swift

```diff
@@ -51,9 +51,7 @@ internal class ProductRepository: ProductRepositoryProtocol {
     }
     
     internal func getProductAppcValue(product: Product, completion: @escaping (Result<String, BillingError>) -> Void) {
-        
-        if let fromCurrency = Coin(rawValue: product.priceCurrency) {
-            billingService.convertCurrency(money: product.priceValue, fromCurrency: fromCurrency, toCurrency: .APPC) {
+        billingService.convertCurrency(money: product.priceValue, fromCurrency: product.priceCurrency, toCurrency: "APPC") {
```

Comment:
Use Currency to represent currencies, not String.

Reply:
Done